app/controllers/exhibits_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ExhibitsController < Spotlight::ExhibitsController
   prepend_before_action :find_exhibit
+  after_action :ingest_members, only: :create
+
+  def ingest_members
+    return if params['exhibit']['slug'].blank?
+    collection_manifest = CollectionManifest.find_by_slug(params['exhibit']['slug'])
+    IIIFIngestJob.new.perform members_of(collection_manifest.id), @exhibit
+  end
+
+  def members_of(manifest_url)
```

Comment:
Should just be able to make this collection_manifest.manifests.map{|x| x['@id']}